src/frontend/components/info-panel/info-panel.ts

```diff
@@ -31,25 +31,8 @@ export class InfoPanel {
 
   constructor(
     private componentDataStore: ComponentDataStore,
-    private userActions: UserActions,
-    private _ngZone: NgZone,
-    @Inject(ElementRef) private elementRef: ElementRef
-  ) {
-
-    this.componentDataStore.dataStream
```

Comment:
@sumitarora Just confirming there's no reason why we need to care about the component store stream here right? Since we're already subscribed to it in `frontend.ts`.